webpack-test.config.js

```diff
@@ -0,0 +1,52 @@
+const webpack = require( 'webpack' );
```

Comment:
I would merge this file with karma.conf.js. There is no point to have separate file which is used only in karma.